QuickGuidePanels/QuickGuidePanels.module.less

```diff
@@ -0,0 +1,39 @@
+// QuickGuidePanels.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/variables.less";
+
+.quickGuidePanels {
+	&.noSteps {
+		.steps {
+			visibility: hidden;
+		}
+	}
```

Comment:
Do we need this code in QuickGuidePanels?

Reply:
We don't need this code because there's no noSteps prop in QuickGuidePanels.
I deleted this code.